desktop/core/src/desktop/js/parse/jison/sql/phoenix/structure.json

```diff
@@ -53,7 +53,7 @@
     "../generic/use/use.jison",
     "../generic/sql_error.jison",
     "../generic/sql_main.jison",
-    "quoted_table_identifier.jison",
+    "../calcite/quoted_table_identifier.jison",
```

Comment:
How about deleting the local unused ones now?

Reply:
So how do know which files are not useful?